website/content/docs/operations/metrics.mdx

```diff
@@ -0,0 +1,98 @@
+---
+layout: docs
+page_title: Boundary Metrics
+description: |-
+  Obtain visibility of various components of a running Boundary
+---
+
+## Boundary Metrics
+
+Boundary provides metrics through the `/metrics` path using a listener with the
+`"ops"` purpose. By default, a listener with that purpose runs on port `9203`.
+See the [example configuration](#example-configuration) section for an example
+listener stanza in a `config.hcl` file.
+
+Metrics are important for understanding what is happening in a boundary
+deployment, enabling diagnoses of deployed controllers and workers. The exported
+metrics grant greater visibility into the various components of a running
+Boundary deployment through Prometheus.
+
+The metrics uses the [OpenMetric
+exposition](https://prometheus.io/docs/instrumenting/exposition_formats/#openmetrics-text-format)
+format which can be read by
+[Prometheus](https://prometheus.io/docs/introduction/overview/). 
+
+### Controller
+
+The following lists the current names and descriptions of metrics provided for controllers.
```

Comment:
nit: how about "The following table lists...."

Reply:
Fixed ([commit](https://github.com/hashicorp/boundary/pull/2024/commits/1c24d17521cbcfdd6892118dc23e8019544b4a53))